.changeset/gold-singers-admire.md

```diff
@@ -0,0 +1,8 @@
+---
+"@espressive/cascara": patch
+"@espressive/design-tokens": patch
+"@espressive/icons": patch
+"@espressive/eslint-config": patch
+---
```

Comment:
I thinnk we do not need a changeset for this, since it touches docs.

Reply:
Correct. @rociobaena remember, a Changeset should only be added if we are changing code that is being shipped. This is docs and is not consumed code.